File: architecture/H1/chapter11/examples_files/OutputFileASCII.java
import java.io.*;

public class OutputFileASCII
{
    public static void main(String[] args) throws IOException {
        PrintWriter outStream =
		new PrintWriter(new FileOutputStream("outfile1.txt"));

	short x = 10;
	outStream.print(x);	// contents of x converted to ASCII decimal
	outStream.close();
    }
}

File: architecture/H1/chapter11/examples_files/OutputFileASCII.cpp
#include <fstream>

using namespace std;

int main() 
{
    ofstream outStream;
    outStream.open("outfile3");
    short x = 10;
    outStream << x;
    outStream.write((char *)&x, sizeof(x));
    outStream.close();

    return 0;
}
